### ISSUE TITLE
App Mobile Menu

### ISSUE THREAD

**Issue by DavideSegullo** [bitsongofficial/sinfonia-ui#403]
The current mobile menu doesn't fit the style of the site and has some UX issues, we can redesign it? Maybe we can use a tab bar navigation like a mobile app (for example like cosmonautico)?

**Comment by Giorgionocera** [comment]
Yeah, I agree with you. C'mon, @giannnni let's try to reinvent this flow!

**Comment by giannnni** [comment]
Hi guys, I designed for you three different approach of mobile menu:
• one with a section label with a dropdown that opens the menu
![image](https://user-images.githubusercontent.com/92384659/180456573-6b14ef65-b441-473c-8670-2ca48d23ed61.png)

• one with a tab style navigation between section
![image](https://user-images.githubusercontent.com/92384659/180456608-880515ff-c935-4bc6-8c50-f5c262f2cc91.png)

• one with the same approach of cosmonautico
![image](https://user-images.githubusercontent.com/92384659/180456626-b704fce3-ffd3-49a5-8fd2-187fbffada75.png)

Do your considerations and feel free to scrap all this proposals.

**Comment by DavideSegullo** [comment]
I like the last one, what do you think @Giorgionocera ?

**Comment by Giorgionocera** [comment]
I agree with @DavideSegullo , I like the third version:
![image](https://user-images.githubusercontent.com/13519292/180467618-07313d58-1c8c-478c-9b7f-c78f47d9df8c.png)

So @DavideSegullo we can implement it 😄
